app/views/tools/saml_request.html.erb

```diff
@@ -0,0 +1,69 @@
+<h1>Validate SAML Authentication Request Signature</h1>
+
+<% if @validation_attempted %>
+  <hr/>
+
+  <div class="usa-alert <%= @request.valid ? "usa-alert--success" : "usa-alert--error"%>">
+    <div class="usa-alert__body">
+      <h2 class="usa-alert__heading">
+        Request: <%= @request.valid ? "Valid" : "Invalid" %>
+      </h2>
+      <% if !@request.valid %>
+        <p>
+          We could not decode your SAML request. Make sure it is encoded correctly, as described in
+          <a href="https://developers.login.gov/saml/#auth-request" target="_blank">our documentation</a>.<br />
+          Also note that all other URL parameters need to be properly URL-encoded.
```

Comment:
Maybe a link to a reputable site that describes what URL encoding is?

Reply:
great idea. do you think wikipedia is ok? that feels very neutral.

Reply:
added in https://github.com/18F/identity-dashboard/pull/691/commits/7d98210d75319007316f4f24596c8c6419595d5a

Reply:
yeah wikipedia works, perfect 🎉